README.md

```diff
@@ -48,6 +48,7 @@ People
 
 #### Students
 <!-- Please keep in Alpha order -->
+* [Joe Abi Sleiman](people/joe-abi-sleiman.md) - jba68 - [http://jas-startup-systems.s3-website.us-east-2.amazonaws.com/grading.html](http://jas-startup-systems.s3-website.us-east-2.amazonaws.com/grading.html)
```

Comment:
Please keep names in alphabetical order

Reply:
Hi Sir,

My last name is Abi Sleiman, which is why I placed it before Agarwal. Am I correct to assume that the alphabetical order is based on last names?

Thanks,
Joe

Reply:
Ok!  I thought Abi == middle name